### ISSUE TITLE
MyAvatar.clearJointsData() / MyAVatar.clearJointData(<join>) Do not sync with other agents

### ISSUE THREAD

**Issue by Menithal** [highfidelity/hifi#9470]
If you deform an avatar using MyAvatar.setJointTranslation / MyAvatar.setJointRotation, Using MyAvatar.clearJointsData or MyAvatar.clearJointData allows one to reset the joint manipulation and it will locally.

However, for everyone else observing, it doesn't seem to pass in the information, and no one else syncs up

```
MyAvatar.setJointTranslation( MyAvatar.getJointIndex("Head"), {x:0, y:-25, z: 30});
Script.setTimeout( function () { MyAvatar.clearJointsData() }, 5000);
``` 

One has to change skeleton URL, to rest the joints

**Comment by PhilipRosedale** [comment]
Thanks - entered as bug #2888

**Comment by hyperlogic** [comment]
I've confirmed that this works in v0.73.0.  This was possibly fixed by PR #12909.  Where we identified and fixed a problem with our avatar mixer protocol.
